src/universal/modules/outcomeCard/components/OutcomeCardAssignMenu/OutcomeCardAssignMenu.js

```diff
@@ -136,12 +159,26 @@ OutcomeCardAssignMenu.propTypes = {
   assignRef: PropTypes.instanceOf(Element),
   atmosphere: PropTypes.object.isRequired,
   closePortal: PropTypes.func.isRequired,
+  styles: PropTypes.object.isRequired,
   task: PropTypes.object.isRequired,
   viewer: PropTypes.object.isRequired
 };
 
+const styleThunk = () => ({
+  label: {
```

Comment:
@dan-f did you borrow these label styles from another menu component? 👍

Reply:
Yeah, I straight-up copy-pasted them from the `Menu` component. For what it's worth, this menu is already skirting around re-using the `Menu` component. Thoughts below...

Reply:
No problem. I think that if components start to get inconsistent we can scope those styles to a shared object and just `{...menuLabelStyle}` on whatever style object we need.